### ISSUE TITLE
ERROR:ipc_channel_reader.cc(332)] IPC message is too big: 136572700

### ISSUE THREAD

**Issue by aep** [byteball/byteball#220]
crashes about 10 seconds after opening when having previously selected full sync

```
ERROR:ipc_channel_reader.cc(332)] IPC message is too big: 136572700
```

```
uname -a
Linux stark 4.11.9-1-ARCH #1 SMP PREEMPT Wed Jul 5 18:23:08 CEST 2017 x86_64 GNU/Linux

```

**Comment by tonyofbyteball** [comment]
fixed by https://github.com/byteball/byteballcore/commit/b1e5634f05c239358692f3582572861ada78a420, the fix will be included in the next release.

If there are no funds on the wallet, it is recommended to delete all app data, restart the app, and choose the option without full download.

**Comment by caffeinebounce** [comment]
I have the same issue, when do you think the next release will be available?

Thanks

**Comment by tonyofbyteball** [comment]
fixed in 1.11.0
